governance-exercises/Le Bateleur - Poll Exercise 02/Course of Action.md

```diff
@@ -0,0 +1,61 @@
+
+Two instances of governance practice precedent exist. (See [1](http://forum.makerdao.com/t/governance-scope-actions-pertaining-to-delegates-in-light-of-mip102c2-sp7-approval/20946), [2](http://forum.makerdao.com/t/notice-aligned-delegate-derecognition-and-avc-member-warning/21099).) Both times, the Governance Facilitator unilaterally acted to derecognize the Aligned Delegates in question; no Governance Poll was involved. In both instances, the identity/whereabouts of the Aligned Delegates was confirmed or strongly suspected; and the Aligned Delegates did not dispute the derecognition process. 
+
+There is also the matter of a whistleblower and how the breach-of-anonymity evidence was presented. In the first of the two instances, the identities of the Aligned Delegates were already known, since the delegation system predated Endgame's anonymity requirement. In the second instance, there appears to have been no whistleblower; and the breach-of-anonymity evidence was investigated privately by the Governance Facilitator.
+
+The immediate scenario presents differently. Diane posted the materials publicly on the Forum. Clive is challenging the verity of the evidence. Critically, it is unclear whether the breach-of-anonymity evidence submitted by Diane rises to the threshold specified in [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy): "If there’s **clear evidence or significant suspicion** [emphasis added] that the identity of an AD has been revealed, a FacilitatorDAO must immediately derecognize the AD." 
+
+The language in Atlas 2.6.6 is ambiguous. While "clear evidence" presents no issue, the phrase "significant suspicion" presents a gray area. Is "reasonable suspicion" equivalent to the "significant suspicion" threshold that we find in Atlas 2.6.6? There's no clear answer based on the text alone. But I am also aware of my [mandate](https://mips.makerdao.com/mips/details/MIP999#2-4-2-alignment-conservers-eligibility-requirements) to minimize risk to Maker and guard against misalignment threats. This is a close call, especially given the ambiguity in Atlas 2.6.6. I am compelled in this instance to err on the side of caution and determine that the materials provided by Diane *do* meet the specified threshold of "significant suspicion."
```

Comment:
Yeah, a fair interpretation. The 'err on the side of caution' wording is fairly compelling (and also bad, imo.)